app/controllers/attendances_controller.rb

```diff
@@ -0,0 +1,58 @@
+class AttendancesController < ApplicationController
+  before_action :set_user, :set_group, :authenticate_user!
+  before_action :set_attendance, only: [:show]
+
+  def index
+    if params[:date].present?
+      @attendances = @group.attendances.where(date: params[:date])
+      redirect_to teacher_group_url(@user, @group), alert: "Not attendances for this day." if @attendances.count.zero?
+    else
+      redirect_to teacher_group_url(@user, @group), alert: "Must specify date."
+    end
+  end
+
+  def show; end
+
+  def new
+    @attendance = Attendance.new
+  end
+
+  def create
+    exist = { group: @group, date: Date.today }
```

Comment:
let's use `attendance_data` instead of `exist` so we have a bit more context on what info is this and why we use it in the query bellow

Reply:
You're right! fixed.